parlai/zoo/pretrained_transformers/README.md

```diff
@@ -1,16 +1,16 @@
 This directory contains:
- - pretrained transformers on reddit and wikipedia + toronto books that can be
+ - pretrained transformers on Reddit and Wikipedia + Toronto Books that can be
   used as a base for pretraining
- - pretrained models fine tuned on Convai2 for the bi-encoder and poly-encoder
+ - pretrained models fine tuned on ConvAI2 for the bi-encoder and poly-encoder
    The polyencoder scores 89+ on convai2 valid set and is fast enough to
    interact in real time with 100k candidates (which are provided.)
 
 ## Interacting with a pretrained model on Convai2
 
 Run this command: (assumes your model zoo is in the default ./data/models)
-```bash
+```
 PYTHONPATH=. python examples/interactive.py -m transformer/polyencoder \
```

Comment:
drop the pythonpath bit please. it assumes things weren't installed following the instructions on our site.